### DISCUSSION TITLE
Workaround for problem with rendering mermaid diagrams in pdfs on WSL2

### DISCUSSION THREAD

**Discussion by Rockgecko** [quarto-dev/quarto-cli#1814] [Q&A]
I have been trying to render a .qmd file to pdf after I added a mermaid diagram. 
I am using the Quarto VScode extension (with VScode attached to wsl via the Remote-wsl extension), but also tried rendering from the command line in wsl directly. In both cases I was getting the error: `ERROR: Couldn't find open server`.

I saw the information in the docs [here](https://quarto.org/docs/authoring/diagrams.html#chrome-install) about potentially needing to install chromium using the command `quarto tools install chromium`. Tried this (from wsl command line), but still got the error afterwards. I could not find anything in the puppeteer docs that seemed relevant, but I did find a suggestion elsewhere that installing google-chrome directly via wsl might resolve issues with chromium. I tried this (following the info in [this](https://askubuntu.com/a/1300155) answer) and am now able to render without errors. 

Not sure it's an official solution, but might do as a workaround, so just posting in case anyone else has the same problem.

**Comment by cscheid** [comment]
Thank you for reporting this! It's useful information for us debugging windows issues going forward. (cc @cderv)
